js/scripts.js

```diff
@@ -0,0 +1,103 @@
+
+//min cust per hour
+//max cust per hour
+//ave # of cookies per cust
+
+//be able to add and remove locations from daily projections report
+//easily modify input #s based on day of the week, special events etc
+//var name = store location
+
+//var min customers
+//var max customers
+//var average cookies
+//var store = new store(name, min, max, avg cookies)
+//something to push info to webpage
+//something to add and remove locations
+//a way to input data (later in the week)
+function avgCookie() {
+    return Math.floor(Math.random() *
+    (max - min + 1) + min);
+    return avgCookie;
+}
+
+function timeStamp(){
+  for (var i = 0; i< 20; i++) {
+      var li = document.crateElement('li');
+      li.textContent = timeStamp.avgCookies[i];
+      ul.appendChild(li);
+}
+
+function Store(name, timeStamp, avgCookie){
+  this.name = name;
+  this.timeStamp = timeStamp;
+  this.avgCookie = avgCookie;
+  return this.Store;
+};
+
+var pikeStore = new Store('Pike' + timeStamp + ' hours' + avgCookie + ' cookies');
```

Comment:
Instead of creating new instances of each store, the goal was to create 5 separate object literals(you will need this code for Tuesdays assignment!)